cmd/generate/config/rules/django.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"github.com/zricethezav/gitleaks/v8/regexp"
+)
+
+func DjangoInsecureSecret() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "django-insecure-secret-key",
+		Description: "Detected a Django insecure secret key, potentially compromising Django’s security protections against privilege escalation and remote code execution.",
+		Regex:       regexp.MustCompile(`["'](django-insecure-[^"']{1,250})["']`),
+		Path:        regexp.MustCompile(`(?i)\.py$`),
```

Comment:
What if we have keys not bordered by `'"`, such as `SECRET_KEY=django-insecure-...`? I'm thinking something like this could be better:
```suggestion
		Regex:       regexp.MustCompile(`\b(django-insecure-[^"'\s]{1,250})["'\s]`),
```

And then we could also detect other files, not just .py